tools/vsimporter/xib2nib/UILabel.cpp

```diff
@@ -153,6 +188,10 @@ void UILabel::ConvertStaticMappings(NIBWriter* writer, XIBObject* obj) {
         AddInt(writer, "UINumberOfLines", _numberOfLines);
     }
 
+    if (_lineBreakMode != 4) {
```

Comment:
as much as I like to do that, but only define 4 as a const didn't really make the readability better because we still have other values like 0/1/2/3 for linkBreak in UILabel. and the same property can be used elsewhere, like UIButton. What should happen is that we define enums like UIKit implementation does so that meaningful enums can be used across the board. but this is out of scope of this change. let me know if you disagree.

---
In reply to: [96054766](https://github.com/Microsoft/WinObjC/pull/1701#discussion_r96054766) [](ancestors = 96054766)